centreon/src/Core/Notification/Application/UseCase/AddNotification/Factory/NotificationResourceFactory.php

```diff
@@ -63,15 +61,7 @@ public function __construct(
     public function create(NotificationResourceRepositoryInterface $repository, array $resource): NotificationResource
     {
         $resourceIds = array_unique($resource['ids']);
-
-        if ($this->user->isAdmin()) {
-            // Assert IDs validity without ACLs
-            $existingResources = $repository->exist($resourceIds);
-        } else {
-            // Assert IDs validity with ACLs
-            $accessGroups = $this->readAccessGroupRepository->findByContact($this->user);
-            $existingResources = $repository->existByAccessGroups($resourceIds, $accessGroups);
-        }
```

Comment:
Is this has been validated by the product ? It's seems odd to not filter resources by Access Group for a non admin user, that mean any user has access to any resources

Reply:
> Is this has been validated by the product ? It's seems odd to not filter resources by Access Group for a non admin user, that mean any user has access to any resources

"Possible to save any BV with any ACL Groups to any contact & contact groups" 
This is what is asked in the ticket and its not yet validated by the product

Reply:
Before requesting a revision, you must first have your modifications validated.